easybuild/easyconfigs/b/bwa-mem2/bwa-mem2-2.2.1-intel-compilers-2023.1.0.eb

```
@@ -15,10 +15,14 @@ toolchainopts = {'pic': True, 'oneapi': False}
 github_account = 'bwa-mem2'
 source_urls = [GITHUB_SOURCE]
 sources = ['v%(version)s.tar.gz']
-patches = ['%(name)s-%(version)s_use_external_deps.patch']
+patches = [
+    'bwa-mem2-%(version)s_use_external_deps.patch',
+    'bwa-mem2-%(version)s_use_common_avx512flags.patch',
```

```suggestion
    'bwa-mem2-%(version)s_common_avx512flags.patch',
```
Or rename the patch to be consistent with the reference here.

---

@verdurin thanks! I'm dumb :smile: